### PR TITLE
Current code

### PR DESCRIPTION
**Pull request by kiiimju**

### CODE REVIEW

File: spec/concepts/todo/create_spec.rb [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 RSpec.describe Todo::Create do
   it "can be spec'd" do
-    raise "Implement me"
+    todo = Todo::Create.(todo: {title: "Sample title"}).model
+    expect(todo.title).to eq("Sample title")
   end
```
Comment (kennethcalamay) [inline spec/concepts/todo/create_spec.rb]: Hi Chanel, can we describe what Todo::Create does using this spec file?

e.g.
- what happens when a name is specified?
- what happens when it's not?

File: app/concepts/todo/create.rb [hunk 2]
```diff
@@ -10,41 +12,49 @@ class Todo::Create < Trailblazer::Operation
...
+
   def process(params)
     # make sure title is not blank
-    return invalid! if params[:todo][:title].blank?
-
     # build the todo
-    todo = @model = Todo.new(params.require(:todo).permit(:title, :description))
+    validate(params[:todo]) do
+      contract.save
+    end
```
Comment (kennethcalamay) [inline app/concepts/todo/create.rb:12]: Looks good. Good start with this one.

File: app/concepts/todo_list/create.rb [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+        end
+
+      # assign the todo_list to the user
+        model.user = current_user
+        model.save!
+    else
+      model = TodoList.find(params[:todo_list_id])
+    end
+  end
+end
```
Comment (kennethcalamay) [inline app/concepts/todo_list/create.rb]: This is a great start for TodoList::Create. User `@model =` for this iteration to make everything work.

File: spec/concepts/todo/create_spec.rb (hunk 1 above)
Comment (kiiimju) [inline spec/concepts/todo/create_spec.rb]: Hi sir,
1. You mean title sir? If so, if the title is specified, like for example is "Sample title", the RSpec expects to see the title of the todo to be "Sample title".
2.  If there is no title passed or the title is not equivalent to "Sample title", but the RSpec is still expecting the title, then the test will fail.

Comment (kennethcalamay) [inline spec/concepts/todo/create_spec.rb]: In RSpec we could make use of contexts like so:

```
describe Something do
    context "when a name is given" do
        it "creates a new something" do
            something = Something.new
            something.name = "My name"
            something.save
            expect(something).to be_persisted
        end
    end

    context "when a name is not given" do
        it "does not create a new something" do
            something = Something.new
            something.name = nil
            something.save
            expect(something).to_not be_persisted
        end
    end
end
```

**Pull request closed without merging**